### DISCUSSION TITLE
Setting up Tracing for bytewax jobs

### DISCUSSION THREAD

**Discussion by shubhanshu02** [bytewax/bytewax#530] [Q&A]
Is there any documentation for setting up tracing for Bytewax jobs?

All I could find is https://docs.bytewax.io/stable/api/bytewax/bytewax.tracing.html, but not sure how to setup the tracing environments. Any example would be really helpful.

**Comment by Psykopear** [comment]
Hi, I moved this into a discussion as it's not properly an issue.
There is a chapter of our guide that goes over an example setup, here: https://docs.bytewax.io/stable/guide/deployment/metrics.html

Let me know if that's enough to get started or if you have more questions
